degica/komoju-woocommerce
# Add lint (php-cs-fixer) & CI action

This is the first step of adding a testing/CI setup to this project.

This PR introduces php-cs-fixer as a PHP linter. It also includes the autofixes from this lint.
Ruleset used is the Symfony ruleset matching the configuration of https://github.com/degica/komoju-php

## How to test
1. Observe the github action and confirm proper operation
2. Review the style changes
